test/util/Aliases.ts

```diff
@@ -1,70 +1,72 @@
 import type * as RDF from 'rdf-js';
 import { stringToTerm } from 'rdf-string';
 
-import type { GeneralEvaluationConfig } from './generalEvaluation';
-import type { IEvaluationConfig, Notation } from './TruthTable';
+/**
+ * Maps short strings to longer RDF term-literals for easy use in making evaluation tables.
+ * Ex: { 'true': '"true"^^xsd:boolean' }
+  */
```

Comment:
Formatting. Also, some off these functions could use a line of documentation or an example (especially the latter).

Reply:
I don't think I understand what you mean.

Reply:
Ah it's outdated now, but there was a space missing.

For the rest, these utility functions (below, I was lazy and bundled my feedback) really benefit from having a comment that provides an example of input/output. Since this is becoming a general test cleanup PR anyway, I thought it would make sense (but it's not required, just nice to have, so as you wish).

Reply:
Well I added some explanation in the last commit so should be fine now anyways :D